### PR TITLE
Ticket5

### PR DESCRIPTION
**Pull request by bashirzadehdanial**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -15,6 +19,11 @@ function App() {
      <Routes>
        <Route path='/articles'  element={<Articles/>} />
        <Route path='/articles/:id'  element={<Article/>} />
+       <Route path='/topics' element={<Topics/>} />
+       <Route path='/topics/coding' element={<CodingTopic />} />
+       <Route path='/topics/football' element={<FootballTopic />} />
+       <Route path='/topics/cooking' element={<CookingTopic />} />
```
Comment (emilyb93) [inline src/App.js]: We should not be hardcoding pages that are all set up the same. topics/coding football and cooking should all be one single route and one single element using a parametric path and useParams like this;

\<Route path='/topics/:topic_name' element={\<TopicPage>}> />

**Review by emilyb93 — CHANGES_REQUESTED** [review]
Refactor your pages so that you only use one page for listing out the topics for each of the topics. The page is the same for each, we should just fetch different data based on the page

**Review by DCrawley94 — CHANGES_REQUESTED** [review]
You're almost there on this one but notice how similar the `articles` and `parametricTopics` components are. You could combine this into a single component which would be much tidier!

Also I notice you're filtering the articles on the front end. Try to avoid this, you have queries on the backend that can do all the hard work for you!

**Review by Augs0 — APPROVED** [review]
Well done, Danial. This is looking good now :)

**Pull request merged**